server/utils/aggregation.js

```
@@ -0,0 +1,144 @@
+    const date = new Date(event.created_at);
+    let key;
+
+    switch (period) {
+      case 'hour':
+        key = `${date.toISOString().split('T')[0]} ${date.getHours()}:00`;
+        break;
+      case 'day':
+        key = date.toISOString().split('T')[0];
+        break;
+      case 'week':
+        const weekStart = new Date(date);
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)